include/aws/testing/aws_test_harness.h

```diff
@@ -434,7 +449,7 @@ static inline int s_aws_run_test_case(struct aws_test_harness *harness) {
         test_res |= harness->on_after(allocator, setup_res, harness->ctx);
     }
 
-    if (test_res != AWS_OP_SUCCESS) {
+    if (test_res == AWS_OP_ERR) {
```

Comment:
should we define anything outside success and skip as error? its corner case, but if test returns something weird outside the defined constants we should probably fail